R/MltplxExperiment.R

```diff
@@ -179,24 +202,27 @@ as_MltplxExperiment.list = function(l, ...){
   unique_classes = unlist(list_classes) %>% unique()
   if(any(n_classes > 1) ||
     length(unique_classes) > 1 ||
-    unique_classes[1] != "MltplxObject"){
+    !(unique_classes[1] %in% c("MltplxObject", "ppp"))){
     stop(paste('to convert "list" object to "MltplxObject" object, all',
-               'elements of list must be of class "MltplxObject"'))
+               'elements of list must either be of class "MltplxObject" or ',
+               'ppp'))
+  }
+
+  if(unique_classes[1] == "ppp"){
+    l = l %>%
+      imap(\(p, i){
```

Comment:
Instead of `imap`, should this use `furrr::future_map` as well?

Reply:
Yup- fixed